README.md

```diff
@@ -714,6 +714,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Matt Oliver](https://github.com/mattyocode)
   - [Matthias Maier](https://github.com/NukeTheFridge)
   - [Max Base](https://github.com/basemax)
+  - [Mayank Mahajan](https://github.com/mayank2002-jpg)
```

Comment:
Please add your name after Mayank Kumar Sharma, in line 720.
Let us know if you need any help doing that! 🙂


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
ok sure.